website/api/blockchain-integration/contracts/verifier.md

```diff
@@ -62,13 +70,19 @@ You can use the [deployed contracts for a released version][doc-released-contrac
 <!-- TODO: Move this example into risc0-ethereum such that it will be under the same version management -->
 
 [doc-released-contracts]: /api/blockchain-integration/contracts/verifier#contract-addresses
+[emergency-stop-mechanism]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/version-management-design.md#emergency-stop
 [EvenNumber.sol]: https://github.com/risc0/risc0-foundry-template/blob/main/contracts/EvenNumber.sol#L46-L52
 [foundry-template]: https://github.com/risc0/risc0-foundry-template
+[function-getVerifier]: https://github.com/risc0/risc0-ethereum/blob/6132e946a142cea6f53f325e8fd0c175ad0ec283/contracts/src/RiscZeroVerifierRouter.sol#L74
 [Groth16Receipt]: https://docs.rs/risc0-zkvm/latest/risc0_zkvm/struct.Groth16Receipt.html
 [IRiscZeroVerifier]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/src/IRiscZeroVerifier.sol
+[proof-composition]: /api/zkvm/composition
+[RISCZeroGroth16Verifier]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/src/groth16/RiscZeroGroth16Verifier.sol
+[RiscZeroVerifierEmergencyStop]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/src/RiscZeroVerifierEmergencyStop.sol
 [term-image-id]: /terminology#image-id
 [term-journal]: /terminology#journal
 [term-receipt]: /terminology#receipt
 [term-verify]: /terminology#verify
 [term-zkvm]: /terminology#zero-knowledge-virtual-machine-zkvm
+[verify-sepolia-etherscan]: https://sepolia.etherscan.io/tx/0x28803b006ac64c08d386676abf34d8ba6232173b28f09ddd4f0c253e37904f91/advanced
```

Comment:
Where does this transaction come from, and what is the contract address its being sent to (https://sepolia.etherscan.io/address/0x35fca632ac537b2b428a864b9d62e45644166b06)? It's not a verified contract, and it doesn't match the deployed verifier address for 1.0 or 1.1.